datasets/wiki_atomic_edits/README.md

```diff
@@ -0,0 +1,147 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- de
+- en
+- es
+- fr
+- it
+- ja
+- ru
+- zh
```

Comment:
since the language depends on the config we need to group the langages by config name
For example
```
languages:
  german_insertions:
  - de
```
could you do this change please ?